zone/client.cpp

```diff
@@ -179,6 +179,14 @@ Client::Client(EQStreamInterface* ieqs)
 	for (int client_filter = 0; client_filter < _FilterCount; client_filter++)
 		ClientFilters[client_filter] = FilterShow;
 
+	m_TimeSinceLastPositionCheck = 0;
+	m_DistanceSinceLastPositionCheck = 0.0f;
+	m_ShadowStepExemption = 0;
+	m_KnockBackExemption = 0;
+	m_PortExemption = 0;
+	m_SenseExemption = 0;
+	m_AssistExemption = 0;
+	m_CheatDetectMoved = false;
```

Comment:
I know the source is inconsistent but I believe the effort is to have variable names snake case where possible for these, like `m_distance_since_last_position_check`, `m_cheat_detect_moved`, etc